src/sync/multichain/multichain.d.ts

```diff
@@ -8,43 +8,37 @@ declare module 'multichain' {
   }
 
   export interface Connection {
-    getAddresses(): Promise<string>
+    getAddresses(): Promise<string[]>
 
-    publish(transaction: Transaction): Promise<void>
```

Comment:
Waarom heb je de Types bij deze functies weggehaald? Is nu impliciet `any`, zou mooi zijn als we de types expliciet kunnen houden als dat kan.

Reply:
De nieuwe library werkt niet meer met een object van argument-name -> argumenten, maar met een array van argumenten (zonder argument namen keys). Daarom had ik de objecten (die eerst rechtstreeks doorgegeven werden aan de library) vervangen door losse argumenten. Dat past beter bij de nieuwe library mijns inziens

Reply:
Het zou wel het mooiste zijn om het expliciet te houden om van de Typescript goodies gebruik te kunnen maken. Voor nu is het prima om het te laten, de functies worden immers alleen gebruikt in `multichain.service.ts`, waarin het wel allemaal getyped is